src/material/schematics/ng-generate/mdc-migration/BUILD.bazel

```diff
@@ -16,6 +16,7 @@ ts_library(
     deps = [
         "//src/cdk/schematics",
         "@npm//@angular-devkit/schematics",
+        "@npm//@bazel/runfiles",
```

Comment:
This should only remain in the test target since the end-consumer will not run with Bazel, and this dep is not available anyway (unless we bundle it)